packages/console/src/onboarding/pages/Congrats/index.tsx

```diff
@@ -21,12 +22,11 @@ const Congrats = () => {
   const theme = useTheme();
   const CongratsImage = theme === AppearanceMode.LightMode ? CongratsImageLight : CongratsImageDark;
   const { update } = useUserOnboardingData();
+  const { navigate, currentTenantId } = useContext(TenantsContext);
 
-  const navigate = useNavigate();
-
-  const enterAdminConsole = async () => {
-    await update({ isOnboardingDone: true });
-    navigate('/');
+  const enterAdminConsole = () => {
+    void update({ isOnboardingDone: true });
+    navigate(currentTenantId);
```

Comment:
i'm thinking about actually we should replace all onboarding routes once it's complete. not a blocker though

Reply:
When the onboarding is completed, all routes about onboarding will be replaced with the admin console routes, using replace seems unnecessary, and also because the `OnboardingApp` will be remove this moment, so we're using the navigate from the `TenantsContext`.